_layouts/navbar.md

```diff
@@ -1,2 +1,5 @@
 * [Home Page](/)
-* [⭐ Star The Repo](https://github.com/open-sauced/intro)
\ No newline at end of file
+* [⭐ Star The Repo](https://github.com/open-sauced/intro)
+* Languages
+    * [:us: English](/)
+    * [:fr: French](/translations/fr/)
```

Comment:
Let's keep this line out of this PR, as this PR should not be specific to some idiom but English.

Reply:
You can have this line in #38